compiler/modules/front.syntax/parse.bal

```diff
@@ -38,11 +38,16 @@ public function scanModulePart(SourceFile file, int partIndex) returns ScannedMo
 public function parseModulePart(ScannedModulePart scanned) returns ModulePart|err:Syntax {
     Tokenizer tok = scanned.tokenizer();
     ModuleLevelDefn[] defns = [];
+    Position startPos = scanned.sourceFile().startPos();
+    Position endPos = scanned.sourceFile().endPos();
     ModulePart part = {
         file: scanned.sourceFile(),
         partIndex: scanned.partIndex,
         defns,
-        importDecls: scanned.importDecls
+        importDecls: scanned.importDecls,
+        // placeholder values
```

Comment:
Why use placeholder values? Construct it with right values to start with.

Reply:
comment removed (they are not placeholder values)